app/services/spree_shopify_importer/data_savers/stock_locations/stock_location_creator.rb

```diff
@@ -0,0 +1,26 @@
+module SpreeShopifyImporter
+  module DataSavers
+    module StockLocations
+      class StockLocationCreator < StockLocationBase
+        def create!
+          Spree::StockLocation.transaction do
+            create_spree_stock_location
+            assign_spree_stock_location_to_data_feed
+          end
+          assign_data_to_spree_stock_items
+        end
+
+        private
+
+        def create_spree_stock_location
+          @spree_stock_location = Spree::StockLocation.new(attributes)
+          @spree_stock_location.save(validate: false)
```

Comment:
I guess we told with you about this save and about how to generate a unique field, do you remember that talk?

Reply:
👍